python/units_neutron.cpp

```diff
@@ -54,7 +82,37 @@ void init_units_neutron(py::module &m) {
              }
            })
       .def(py::self == py::self)
-      .def(py::self != py::self);
+      .def(py::self != py::self)
+      .def("__rmul",
+           [](const units::Unit &self, double scalar) -> core::Variable {
+             using namespace scipp::core;
+             return scalar * self;
+           })
+      .def("__rmul",
+           [](const units::Unit &self, int64_t scalar) -> core::Variable {
+             using namespace scipp::core;
+             return scalar * self;
+           })
```

Comment:
Can you use https://pybind11.readthedocs.io/en/stable/advanced/classes.html#operator-overloading? For example `.def(int64_t() * py::self)`.

Reply:
Would it be the same as defining  "__rmul__" which is not working?

Reply:
Oops, yes of course, bad question.